contracts/staking/.cargo/config

```diff
@@ -0,0 +1,5 @@
+[alias]
+wasm = "build --release --target wasm32-unknown-unknown"
```

Comment:
Can you add `wasm-debug` and test the two build commands from the root README:

------------

**Contracts**

Step 1 (fast checks)

```sh
for contract_dir in contracts/*; do (cd "$contract_dir" && cargo fmt && cargo unit-test && cargo wasm-debug && cargo clippy -- -D warnings && cargo schema) || break; done
```

Step 2 (slower checks)

```sh
for contract_dir in contracts/*; do (cd "$contract_dir" && cargo wasm && cargo integration-test) || break; done
```

Reply:
Done